crates/tinymist/src/actor/typ_server.rs

```diff
@@ -99,17 +79,17 @@ pub struct CompileServerActor<C: Compiler> {
     /// Estimated latest set of shadow files.
     estimated_shadow_files: HashSet<Arc<Path>>,
     /// The latest compiled document.
-    latest_doc: Option<Arc<TypstDocument>>,
+    pub(crate) latest_doc: Option<Arc<TypstDocument>>,
     /// The latest successly compiled document.
     latest_success_doc: Option<Arc<TypstDocument>>,
     /// feature set for compile_once mode.
     once_feature_set: Arc<FeatureSet>,
     /// Shared feature set for watch mode.
     watch_feature_set: Arc<FeatureSet>,
 
-    /// Internal channel for stealing the compiler thread.
-    steal_tx: mpsc::UnboundedSender<Interrupt<Self>>,
-    steal_rx: mpsc::UnboundedReceiver<Interrupt<Self>>,
+    /// Channels for stealing the compiler thread.
```

Comment:
The stealing interrupt only refers to the `Interrupt::Task`.
We can update the document to:
```rs
/// Channels for sending interrupts to the compiler thread.
```